Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -704,6 +704,12 @@ public virtual IEnumerable<string> GetFileSystemEntryPaths(string path, bool rec
             return Directory.EnumerateFileSystemEntries(path, "*", GetEnumerationOptions(recursive));
         }
 
+        /// <inheritdoc />
+        public virtual bool Exists(string path)
+        {
+            return Directory.Exists(path) || File.Exists(path);
```

Comment:
Only the `Directory.Exists(path)` path get used now, no?
Shouldn't we split this up into 2 functions, `DirectoryExists` and `FileExists` or something.
The current code will do 2 syscalls if it the dir doesn't exist or if it's a file (I think)